Lesson16/fercan/script.js

```diff
@@ -0,0 +1,100 @@
+// 3. Write a function that asks the user to enter two numbers and checks if the first number is greater than, less than, or equal to the second number using if else statement.
+const a = "a";
+const b = "b";
+if (a < b) {
+  // true
+  console.log(`${a} is less than ${b}`);
+} else if (a > b) {
+  console.log(`${a} is greater than ${b}`);
+} else {
+  console.log(`${a} and ${b} are equal.`);
+}
+
+// 5. Write a function that takes in a number and checks if it's positive, negative, or zero using if else statement.
+
+function isPositiveOrNegative (number) {
+if(number<0){
+    console.log('It\'s a negative number');
+ } else if (number>0) {  
+    console.log('It\'s a positive number');
+ } else { 
+    console.log ('It\'s equal to zero.');
+}}
+// 6. Write a function that takes in a year and checks if it's a leap year (is divisible by 4 and not divisible by 100 OR divisible by 400) 
+//or not using if else statement.
+
+function isLeapYear (year) {
```

Comment:
The function does not work correctly now as if you use 'else' statement, you cannot provide a condition. https://www.w3schools.com/js/js_if_else.asp

You could do like this:
```
if(year%4==0 && year%100!=0){
        console.log('It\'s a leap year');
     } else if (year%400==0); {  
        console.log('It\'s a leap year');
    } else {
      console.log('It\'s not a leap year');
  }
}
```

or you can do it in a shorter way by combining 2 conditions with logical OR statement (||):
```
if ((year % 4 === 0 && year % 100 !== 0) || year % 400 === 0) {
        return `${year} is leap year`;
    } else {
        return `${year} is not a leap year`;
    }
```